dev/files/killConjurServer

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+kill $(ps aux | grep '[r]uby /usr/local/bin/conjurctl server' | awk '{print $2}')
```

Comment:
Looks like everything else in this folder is in a subdirectory corresponding to the relevant authenticator. Since this is only used in the OIDC authenticator, should we put it in an `authn-oidc` subdir?

Reply:
@jtuttle i thought about that but this isn't really oidc logic and it could be used in other scenarios that need loading variables. That's why i put this in the general folder. wdyt?

Reply:
Yeah okay, that's fine.